src/mitol/openedx/utils/courses.py

```diff
@@ -0,0 +1,24 @@
+"""Utility functions for course related functions"""
```

Comment:
You can also import this function in the `utils/__init__.py` so that we could import it directly from the courses module. This is just nitpick though

Reply:
Sure